subcommand/get-consul-client-ca/command.go

```diff
@@ -0,0 +1,227 @@
+package getconsulclientca
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/command/flags"
```

Comment:
It's used for `c.help = flags.Usage(help, c.flags)`. I can move that over though.

Reply:
On a second look, this might require a more comprehensive review of how to do that (This function looks a bit more complex than I hoped for). It's also used all over the repo and will definitely need to be addressed as part of not importing this library task. It probably doesn't make sense to spearhead it in this PR and feels a bit out of scope.